bubble-wrap-style.yaml

```diff
@@ -4414,13 +4274,14 @@ layers:
                 kind: [enclosure, animal]
             draw:
                 mapzen_icon_library:
-                    priority: 43
                     visible: false
-                    text:
-                        text_wrap: 10
-                        font:
-                            style: italic
-                            weight: normal
+                text:
```

Comment:
Just to confirm, this `text` is supposed to be an alternative to a `mapzen_icon_library` icon, which is turned off? Elsewhere we use that designated `text-blend-order` style for such cases, right?

Reply:
Well, funny you should ask. If I move to `text-blend-order` instead of `text` around 50% of the labels are culled when I expect them to be shown :\

http://localhost:8000/#18/37.73212/-122.50216

![image](https://user-images.githubusercontent.com/853051/62810569-27229180-bab3-11e9-80dc-89f5cb22ac13.png)

Reply:
Well, I don't know why but this other suggestion of yours is the root cause of the missing text:

>-                    size: [[7,[75%,75%]],[12,[100%,100%]],[15,[110%,110%]]]
>+                    size: [[7,[75%]],[12,[100%]],[15,[110%]]]